extensions/sql-migration/src/wizard/sqlSourceConfigurationPage.ts

```diff
@@ -1,161 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
no longer used?

Reply:
this page was replaced by databaseBackupPage a while back